dvc/exceptions.py

```diff
@@ -1,4 +1,3 @@
-"""Exceptions raised by the dvc."""
```

Comment:
Unrelated change, let's not do this

Reply:
Dunnon how this, happened. I've put it back